btr-web/btr-common-components/components/bcros/inputs/CountriesOfCitizenship/Dropdown.vue

```diff
@@ -109,55 +110,80 @@
         </ComboboxOptions>
       </div>
     </Combobox>
+    <div v-if="hasError" class="text-sm text-red-500 pt-2">
```

Comment:
Can you instead of this wrap it in the `UFormGroup` and add a form in `addNew/index` page if its missing it ?
I'll remove then form in the ticket I am working on right now, and will be able to use errors directly.

or you can leave it like this, and I will do this change once I put everything under single form.